src/App.css

```diff
@@ -55,3 +56,67 @@
 .card {
   border-radius: 50%;
 }
+
+.signedIn {
+  background: rgb(34, 193, 195);
+  background: linear-gradient(135deg, rgba(34, 193, 195, 1) 0%, rgba(253, 187, 45, 1) 100%);
+  font-family: "work sans", sans-serif;
+  padding-top: 5%;
+}
+
+.loginform {
+  display: flex;
+  flex-direction: column;
+  max-width: 500px;
+  min-width: 300px;
+  max-height: 700px;
+  width: 30%;
+  height: 60%;
+  margin: 100px auto;
+  background-color: #fff;
+  border-radius: 25px;
+  margin-top: 1%;
+  box-shadow: rgba(50, 50, 93, 0.25) 0 50px 100px -20px, rgba(0, 0, 0, 0.3) 0 30px 60px -30px, rgba(10, 37, 64, 0.35) 0 -2px 6px 0 inset;
+}
+
+.headerTitle {
+  text-align: center;
+  font-family: 'open sans', sans-serif;
```

Comment:
**[REQUIRED]** - Please fix the responsiveness of the application for all screen sizes (tablets).
-Also Fix the sliders on all views, arrow positions.
- Please include a logo in your application
- Also, can you implement a navbar for the mobile app.
- Please style the sideNav according to the given UI.

Reply:
Thank you for your review. We're working on them. will fix them soon.